pkg/plugins/autodiscovery/flux/main.go

```diff
@@ -0,0 +1,168 @@
+package flux
+
+import (
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/plugins/utils/docker"
+	"github.com/updatecli/updatecli/pkg/plugins/utils/version"
+)
+
+var (
+	// defaultFluxFiles specifies accepted Helm chart metadata file name
+	defaultFluxFiles            []string = []string{"*.yaml", "*.yml"}
+	defaultVersionFilterPattern string   = "*"
+	defaultVersionFilterKind    string   = "semver"
+)
+
+// Spec defines the parameters which can be provided to the Flux crawler.
+type Spec struct {
+	/*
```

Comment:
I'm curious about why use multi-line comments like this here?

This is normally written as

```go
// Auths provides a map of registry credentials where the key is the registry URL without scheme
```
If it's split over multiple lines, just add `//` at the start of each one?

Reply:
Multi-line comments was needed for https://github.com/invopop/jsonschema to correctly generate the jsonschema description otherwise it would just pick the last `//` before the struct field

Reply:
Thanks @bigkevmcd for double checking that the syntax `/* */` wasn't required anymore.
I switched to `//` which I also find easier to read